gridsync/streamedlogs.py

```diff
@@ -78,7 +78,7 @@ def get_streamed_log_messages(self):
         """
         :return list[str]: The messages currently in the message buffer.
         """
-        return list(msg.decode("utf-8") for msg in self._buffer)
+        return list(msg.decode("utf-8") for msg in list(self._buffer))
```

Comment:
The outer `list` already consumes the entire generator before this function returns so the inner `list` isn't really making a real difference, as far as I can see?

How does `_buffer` mutate while the outer list is consuming it?  Is it because ... `_buffer` is shared across threads?

Reply:
Oof. Thanks for the correction, @exarkun. When I initially saw this, I failed to remember that `gridsync.gui.debug.DebugExporter` is indeed moving the log-loading call to a `QThread` to prevent UI-blocking... I'll re-open the issue.